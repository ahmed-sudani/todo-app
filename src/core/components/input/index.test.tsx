import { describe, expect, it } from 'vitest'
import Input from '.'

import { render } from '@testing-library/preact'

describe('Input', () => {
  it('should render successfully', () => {
    const { container } = render(<Input />)
    expect(container).toMatchSnapshot()
  })
})
