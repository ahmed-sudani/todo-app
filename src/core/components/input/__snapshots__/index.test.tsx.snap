// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Input > should render successfully 1`] = `
<div>
  <input
    class="input"
    type="text"
  />
</div>
`;
